src/Git/Value/MergeTargetCandidateBranches.php

```diff
@@ -82,7 +82,9 @@ public function newestFutureReleaseBranchAfter(SemVerVersion $version): BranchNa
         $futureReleaseBranch = Vec\filter(
             Vec\reverse($this->sortedBranches),
             static function (BranchName $branch) use ($nextMinor): bool {
-                return $nextMinor->lessThanEqual($branch->targetMinorReleaseVersion());
+                $targetVersion = $branch->targetMinorReleaseVersion();
+
+                return ! $targetVersion->isNewMajorRelease() && $nextMinor->lessThanEqual($targetVersion);
```

Comment:
I didn't see this method used outside of the set default command, so nothing else should be affected.

Reply:
Ok, lemme rephrase... 

After this change:

If only a new **major** branch exists, it will:
1. create a new **minor** branch
2. create a new **minor** milestone
3. switch to that **minor** milestone
4. **NOT** create a merge-up MR? (to be verified - we **DO WANT** a merge-up to the next major!)

Reply:
Why would it not do step 4? If it's doing this already it should continue to do so, at least that it is my intention to not alter that.

Reply:
I did check btw, here is where I see it's using different methods of this class https://github.com/laminas/automatic-releases/blob/5188a07f5d61c32259ef3f3e7f7a5d0349374066/src/Application/Command/CreateMergeUpPullRequest.php#L56 and so shouldn't be affected.

Reply:
Excellent!

I guess we should try it out somewhere then? Do we have a candidate library somewhere?

Reply:
Anything we can improve on `laminas-authentication` to give this a shot then? I'd wait for a patch there, then release here, and finally release over there :+1:

Reply:
This one might be doable:
- https://github.com/laminas/laminas-view/pull/157